CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## [Unreleased]
 
 - Update `buf beta price` with the latest pricing information.
+- Fix issue where `buf generate buf.build/foo/bar --exclude-path some/path` does
```

Comment:
Fix issue where `buf generate --exclude-path` was not properly excluding paths for remote modules.